src/index.html

```diff
@@ -118,10 +121,12 @@ <h2>Libraries</h2>
     <li><a href="https://wcjochem.github.io/sfarrow/index.html">sfarrow</a> (R)</li>
     <li><a href="https://gdal.org/drivers/vector/parquet.html">GDAL/OGR</a> (C++, bindings in several languages)</li>
     <li><a href="https://github.com/JuliaGeo/GeoParquet.jl">GeoParquet.jl</a> (Julia)</li>
-    <li><a href="https://github.com/planetlabs/gpq">gpq</a> (Go, CLI and WASM build for reading/writing GeoParquet)</li>
+    <li><a href="https://github.com/planetlabs/gpq">gpq</a> (Go and WASM)</li>
+    <li><a href="https://github.com/Toblerity/Fiona">Fiona</a> (Python - as of version 1.9.4. Note the GeoParquet driver will only be available if your system's GDAL library links libarrow; fiona wheels on PyPI do not include libarrow as it is rather large.)</li>
```

Comment:
We can add a similar entry for `pyogrio` (https://github.com/geopandas/pyogrio/, alternative python bindings to GDAL; faster for tabular data access)